css/_list.scss

```diff
@@ -180,7 +180,7 @@ margin-left: 30px;
 
 	&:hover{
 		border-left: 3px solid $gray-500;
-		background-color:$gray-300;
+		background-color:$gray087;
```

Comment:
the reason i changed is was because we have different variables for the hover in the wfm-list and angular-ui-tree so let's reverse it, so we use $gray-300 on both places